code/graphics/post_processing.cpp

```diff
@@ -232,7 +253,7 @@ bool gr_lightshafts_enabled()
 		return false;
 	}
 
-	// supernova glare should disable lightshafts
+	// supernova glare should switch to legacy lightshafts
```

Comment:
Regarding lightshafts specifically, this bit of code is due to issue #4152:

> Part of this is due to the introduction of light shafts in the above referenced commit. When I modified gr_lightshafts_enabled() to return false if a supernova was in progress, this almost entirely fixed the problem - the glare behaved as it was supposed to. However there are still two deficiencies:
> 1. Disabling light shafts when a supernova starts could abruptly remove any lightshafts currently on the screen.
> 2. During the letterbox scene, the entire screen faded to white, not just the part inside the letterbox.

Does this affect the phrasing of the comment?

(Also, on reflection, this comment should link to that issue.)

Reply:
Not really.
Rather, depends on the semantics here.
Effectively, FSO has three modes now:
1. No glare at all.
2. Glare realized through lightshafts
3. Glare realized through screen tint.
This new toggle switches between 1 and 2/3.
The old lightshaft toggle switches between 2 and 3.
Except supernovae, which can override the mode to 3, no matter the setting.
This switch will, probably, look somewhat off in any mode other than 3, as discussed in #4152 unless we implement some sort of soft handover, which is out of scope for this PR.
I'll update the comment to refer to said issue, but I don't think we need to otherwise significantly modify the phrasing of the comment

Reply:
Okey dokey.